local/languages/finna/Carousel/en-gb.ini

```diff
@@ -1,7 +1,9 @@
+external_link = "External link."
 First = "First"
 go_to_page = "Go to page %s"
 Last = "Last"
 Next = "Next"
+opens_new_tab = "Opens in a new tab."
```

Comment:
Muualla on käytetty "Open in a new window" -käännöstä. Käytetään samaa kaikkialla. Jos pitää olla tab, niin sitten uusi käännös kaikkialle.